api/nakadi-event-bus-api.yaml

```diff
@@ -1245,6 +1245,112 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /timelines:
+    post:
+      tags:
+        - timelines-api
+      description: |
+        Creates timeline and switches to it.
+        The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
+        to be able to access this endpoint.
+      parameters:
+        - name: timeline_request
+          in: body
+          schema:
+            description: Event type and storage id to be used for timeline creation.
+            type: object
+            properties:
+              event_type:
+                type: string
+              strorage_id:
+                type: string
+          required: true
+      responses:
+        '201':
+          description: New timeline is created and in use.
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /timelines/{id}#1:
+    delete:
+      tags:
+        - timelines-api
+      description: |
+        Deletes a timeline in case it is the first one.
+        Method call is quite dangerous, because it will remove any timeline.
```

Comment:
This somehow contradicts the previous line. In the previous line you say the timeline is deleted in case it is the first one. Here you say it will be deleted no matter what